multitenancy/types.go

```diff
@@ -0,0 +1,180 @@
+package multitenancy
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/jpmorganchase/quorum-security-plugin-sdk-go/proto"
+)
+
+type ContractVisibility string
+type ContractAction string
+
+const (
+	VisibilityPublic  ContractVisibility = "public"
+	VisibilityPrivate ContractVisibility = "private"
+	ActionRead        ContractAction     = "read"
+	ActionWrite       ContractAction     = "write"
+	ActionCreate      ContractAction     = "create"
+
+	// QueryOwnedEOA query parameter is to capture the EOA address
+	// For value transfer, it represents the account owner
+	// For message call, it represents the EOA that signed the contract creation transaction
+	// in other words, the EOA that owns the contract
+	QueryOwnedEOA = "owned.eoa"
+	// QueryToEOA query parameter is to capture the EOA address which is the
+	// target account in value transfer scenarios
+	QueryToEOA = "to.eoa"
+	// QueryFromTM query parameter is to capture the Tessera Public Key
+	// which indicates the sender of a private transaction or participant of a private contract
+	QueryFromTM = "from.tm"
+
+	// AnyEOAAddress represents wild card for EOA address
+	AnyEOAAddress = "0x0"
```

Comment:
`0x0` fits better as it reflects the hex format of an Ethereum account address. Also we don't have to deal with URL encoding ( * = %2a) when having the wild card value in the query string. This would make the scope string more readable.